elastic/migrations/2020_08_25_084243_create_playlists_index.php

```diff
@@ -0,0 +1,32 @@
+<?php
+declare(strict_types=1);
```

Comment:
Need a blank line above this. (Resolve it for other occurrences.

Reply:
Added blank line above declare() in playlists and projects elastic migrations.